### PR TITLE
Angular issue tracker demo app

### PR DESCRIPTION
**Pull request by bshepherdson**

### CODE REVIEW

File: angular/tracker/src/models.js [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+    {
+      name: 'isOpen',
+      value: false
+    }
+  ],
+
+  values: [
+    {
+      name: 'NEW',
+      values: { label: 'New', isOpen: true }
```
Comment (adamvy-google) [inline angular/tracker/src/models.js:47]: The seaprate values:{} field is no longer required for enums.

Just

{
name: 'NEW',
label: 'New',
isOpen: true
}

File: angular/tracker/src/models.js [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+    {
+      name: 'OBSOLETE',
+      values: { label: 'Obsolete' }
+    }
+  ]
+});
+
+
+foam.CLASS({
+  package: 'ng.dao',
```
Comment (adamvy-google) [inline angular/tracker/src/models.js:86]: Some sort of documentation for this class?  What does it do?

**Comment by adamvy-google** [comment]
I think we should move this to com.google.foam.demos packages, and src/com/google/foam/demos/...

It needs to be in a demo directory somewhere, and src/com/google/foam/demos is the canonical one now.

File: angular/tracker/src/models.js [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+});
+
+foam.CLASS({
+  package: 'ng.tracker',
+  name: 'User',
+
+  properties: [
+    { class: 'String', name: 'id', required: true },
+    { class: 'String', name: 'name', required: true },
+    { class: 'String', name: 'imageUrl' }
```
Comment (kgrgreer) [inline angular/tracker/src/models.js:221]: We have URL type.

**Pull request merged**